Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -403,6 +403,7 @@ BoardInit (
     if (FeatureCfgData != NULL && (FeatureCfgData->MeFipsMode != 0)){
       HeciSetFipsMode(FeatureCfgData->MeFipsMode);
       DEBUG ((DEBUG_INFO, "Enabled FIPS mode.\n"));
+      DEBUG ((DEBUG_INFO, "Set HeciSetFipsMode to 0x%x\n", FeatureCfgData->MeFipsMode));
```

Comment:
@tsaikevin : SInce HeciSetFipsMode() does return a status, can we check that to print the message accordingly?

Reply:
Printing debug messages can be independent of the function return status. If you want, you can print the function return status in addition to what gets printed now.

Reply:
This message only prints the config data value when FIPS is enabled.
Lets change it to printing the cfgdata value to satisfy the test case requirement, and then the status later.

Reply:
code is updated, pls review again thanks